lib/suite.js

```diff
@@ -108,6 +110,25 @@ Suite.prototype.timeout = function(ms) {
 };
 
 /**
+ * Set number of times to retry a failed test.
+ *
+ * @api private
+ * @param {number|string} n
+ * @return {Suite|number} for chaining
+ */
+Suite.prototype.retries = function(n) {
+  if (!arguments.length) {
+    return this._retries;
+  }
+  if (n.toString() === '0') {
+    this._retries = -1;
```

Comment:
Seems like `if (n.toString() === '0') this._retries = -1;` is probably unnecessary? If a user did `.retries(-1)`, `.retries(0)`, `.retries(false)` etc, it'd be falsey anyway, as long as you make sure `_currentRetry` starts at 1